tensorflow/contrib/makefile/Makefile

```diff
@@ -910,6 +910,12 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac.
+ifeq ($(TARGET),OSX)
+  ifeq($(HOST_ARCH), x86_64)
+	HOST_LDOPTS += -framework CoreFoundation
+	LIBS += -framework CoreFoundation
+  endif()
```

Comment:
It should be "endif".

Reply:
Sorry, was in CMake mode, fixed.

Reply:
@themightyoarfish 
Even though I applied your patch, I still cannot compile tensorflow successfully.
I attached a makefile for your reference, which is working fine for me. 

+++ b/tensorflow/contrib/makefile/Makefile
@@ -908,6 +908,9 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac.
 ifeq ($(TARGET),IOS)
+  ifeq ($(HOST_ARCH), x86_64)
+    HOST_LDOPTS += -framework CoreFoundation
+  endif
   ifeq ($(IOS_ARCH),X86_64)
     HOST_LDOPTS += -framework CoreFoundation
   endif

Reply:
I'm wondering if we need `IOS_ARCH` if below that

Reply:
With some `AND`ing and `OR`ing it could probably be shortened, but I don't know how exactly.

Reply:
@jekyll88 Your patch seems to apply to iOS compilation only?

Reply:
@themightyoarfish 
Certainly, the change from me is only applied for iOS build.
That's because I only build iOS package by the command below.

`tensorflow/contrib/makefile/build_all_ios.sh -a arm64`

Reply:
Let's first get this macOS fix merged, can you open a separate PR based on the merge that will fix it for iOS?

Reply:
@themightyoarfish Sure, a PR #30519 already was created. I will update it if any merge conflict.